src/app/components/repertoire/repertoire.component.ts

```diff
@@ -52,12 +78,18 @@ export class RepertoireComponent implements OnInit {
 
     getSeasonPerformancesSub$
       .subscribe((performances) => {
-        this.perfomances = performances;
-        this.collectionSize = performances.length;
-        this.changeDetector.markForCheck();
-        this.loaderService.stop('repertoire');
-      },
-      error1 => this.loaderService.stop('repertoire')
-    );
+          this.performances = performances;
+          this.collectionSize = performances.length;
+          this.changeDetector.markForCheck();
+          this.loaderService.stop('repertoire');
+        },
+        error1 => this.loaderService.stop('repertoire')
```

Comment:
Why error1? Why not error2, error3, error564623423543534?

Reply:
Fixed